SECURITY.md

```diff
@@ -0,0 +1,21 @@
+# Security Policy
+
+## Supported Versions
+
+Use this section to tell people about which versions of your project are
+currently being supported with security updates.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 5.1.x   | :white_check_mark: |
+| 5.0.x   | :x:                |
+| 4.0.x   | :white_check_mark: |
+| < 4.0   | :x:                |
```

Comment:
These don't really look correct?

Reply:
It's more like a template, like we have the issue template. So there's some guidance to add a security vulnerability.

Reply:
Not really, it's the security policy of our project so that users can see what our security support model is.

For example https://github.com/tomkerkhove/promitor/security and shows up in the issue template chooser:
![image](https://user-images.githubusercontent.com/4345663/107737865-b7274400-6d05-11eb-8f93-54f8f1af7ece.png)


If this is for reporting security vulnerabilities, we should use an issue template instead similar to https://github.com/tomkerkhove/promitor/issues/new?assignees=tomkerkhove&labels=security&template=3_security_issue.yml (this is using a rendered issue preview so you can ignore that, but you get the point)

Reply:
Aha, OK, we should consider adding both then, but then I'm not sure how this version table works exactly.

Reply:
If we want to yes, but it's a commitment.

The table says which versions we will patch in case of security vulnerabilities.